src/bp-forums/groups.php

```diff
@@ -252,6 +261,45 @@ public function remove_group_forum_meta_cap_map() {
 
 		/** Edit ******************************************************************/
 
+		/**
+		 * Disabled dropdown options for forum.
+		 *
+		 * @since BuddyBoss 1.7.6
+		 *
+		 * @param string $attr_output Default attributes.
+		 * @param object $object      Froum post data.
+		 * @param array  $args        Froum dropdown arguments.
+		 *
+		 * @uses bbp_get_forum_post_type() Forum post type.
+		 * @uses bbp_get_forum_group_ids() Get forum group id.
+		 *
+		 * @return string
+		 */
+		public function disabled_forum_dropdown_options( $attr_output, $object, $args ) {
```

Comment:
move this function to the end of the file.

Reply:
Move the function at the end of the file.
https://github.com/buddyboss/buddyboss-platform/pull/2117/commits/b2c663fb81f0630d02690c0ac5064d62ff9f8387